e2e/volumegroupsnapshot_base.go

```diff
@@ -456,6 +456,22 @@ func (v *volumeGroupSnapshotterBase) testVolumeGroupSnapshot(vol VolumeGroupSnap
 		return fmt.Errorf("failed to create volume group snapshot: %w", err)
 	}
 
+	// Create and delete 5 additional group snapshots to test flattening on minSnapshotLimit
```

Comment:
This is a generic base for both cephfs and rbd, IMO this either needs a separate case or needs adjusted where cephfs is not impacted because of this one.

Reply:
Removed rbd specific comment.
I think creation & deletion of multiple volgroupsnap serves a good default test case for cephfs too.